.gitignore

```diff
@@ -1,14 +1,5 @@
 # Files and directories created by pub
-.buildlog
+.dart_tool
 .packages
-.project
 .pub
```

Comment:
this isn't created anymore is it?

Reply:
It is a VERY old thing from Dart Editor days.